docs/tendermint-core/using-tendermint.md

```diff
@@ -183,7 +183,7 @@ endpoints. Some take no arguments (like `/status`), while others specify
 the argument name and use `_` as a placeholder.
 
 
-> TIP: Find the RPC Documentation [here](https://docs.tendermint.com/master/rpc/)
+> TIP: Find the RPC Documentation [here](https://docs.tendermint.com/main/rpc/)
```

Comment:
It should be nice if https://docs.tendermint.com/rpc/ was the address of this documentation, even though under the hood if redirected to [master/main/latest]/rpc/

Reply:
It's not clear whether our theme actually supports this right now.